opentelemetry-exporter-gcp-trace/tests/test_cloud_trace_exporter.py

```diff
@@ -621,6 +622,54 @@ def test_too_many_link_attributes(self):
     def test_extract_empty_resources(self):
         self.assertEqual(_extract_resources(Resource.get_empty()), {})
 
+    def test_extract_resource_attributes_with_regex(self):
+        resource_regex = re.compile(r"service\..*")
```

Comment:
I notice we are using `regex.match()` above, in which case the `.*` isn't required here. What do you think is more useful/intuitive, `fullmatch()` or `match()`?

Reply:
I personally don't use python regexes that often, so I wouldn't read too much into my intention based on this regex (i.e. this is probably an inefficient / "bad" regex because I don't write them that often).  As a general rule, I would use which ever one seems more prevalent.  But I would certainly defer to you as to what you prefer on this one.

Reply:
I think the same sort of thing could be accomplished with all of them. Lets leave it as match for now 👍